### PR TITLE
Reduce stacktrace of VendorListService

### PR DESCRIPTION
**Pull request by AndriyPavlyuk**
The logs of VendorListService were reduced to short message.

### CODE REVIEW

File: src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java [hunk 1]
```diff
@@ -286,7 +286,7 @@ private static boolean isValid(VendorList vendorList) {
      * Handles errors occurred while HTTP request or response processing.
      */
     private static Future<Void> failResponse(Throwable exception, int version) {
-        logger.warn("Error fetching vendor list via HTTP for version {0}", exception, version);
+        logger.warn("Error fetching vendor list via HTTP for version {0}", version);
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java]: i'd like to see more details when error occurred.
Also, pls look at PR https://github.com/rubicon-project/prebid-server-java/pull/574 which adds INFO error level logging.

Comment (AndriyPavlyuk) [reply src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java]: Ok

File: src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java [hunk 2]
```diff
@@ -286,7 +286,9 @@ private static boolean isValid(VendorList vendorList) {
      * Handles errors occurred while HTTP request or response processing.
      */
     private static Future<Void> failResponse(Throwable exception, int version) {
-        logger.warn("Error fetching vendor list via HTTP for version {0}", exception, version);
+        logger.warn("Error fetching vendor list via HTTP for version {0} with message: {1}",
+                version, exception.getMessage());
+        logger.debug("Error fetching vendor list via HTTP {0}", version, exception);
```
Comment (rpanchyk) [inline src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java]: Seems you've wrong sequence of `version, exception` arguments order.

Comment (AndriyPavlyuk) [reply src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java]: I'll change it.

**Pull request merged**